xbmc/video/VideoThumbLoader.cpp

```diff
@@ -397,7 +397,8 @@ bool CVideoThumbLoader::FillLibraryArt(CFileItem &item)
   {
     m_videoDatabase->Open();
 
-    if (tag.GetAssetInfo().GetId() >= 0)
+    const CURL itemUrl{item.GetPath()};
+    if (itemUrl.HasOption("videoversionid"))
```

Comment:
Why this change? Just want to understand...

Reply:
This is how the code knows that we're browsing movies in the library and considering the movies as groups of versions (use "movie" art type) or using one of the paths that displays the individual versions (use "videoversion" art type): versions in folder, choose version dialog, versions navigation, versions manager dialog.

Same could be done with a GetPath() and looking for a videodb://movies/videoversions/ prefix.

It's not forward looking but works today. In v22 this will have to be redone completely, stuff moved around, ...

Reply:
Okay, thanks for the explanation.

Reply:
refactored into a method of CFileItem as the same condition will have to be used to fix stuff in info dialog / context menu opened on versions, if the concept of this PR is accepted.